<ui version="4.0" >
 <class>ECCDTRANSLINE_0</class>
 <widget class="QWidget" name="ECCDTRANSLINE_0" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>828</height>
   </rect>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Transmission line</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>50</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor" >
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text" >
    <string>W/G S/W, RF/GV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>90</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor" >
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text" >
    <string>Vacuunm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>130</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor" >
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text" >
    <string>Polarization</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>170</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor" >
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text" >
    <string>Temperature</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>210</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor" >
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text" >
    <string>Dummy2 Temp.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
